tsconfig.json

```diff
@@ -9,13 +9,10 @@
     "lib": ["es2017", "dom"],
     "module": "commonjs",
     "moduleResolution": "node",
-    "target": "es2017",
+    "target": "es6",
```

Comment:
Please revert.

Reply:
While I have reverted the reason for this change was that `package.json` says it supports `node >= 6` ... using `es2017` means we can't support Node v6 as `async` & `await` doesn't work with target set to `es2017`. I'd say we move to support only `node >=8` (which will be the official support target for LBNext).